_author_instructions/01_policies.md

```diff
@@ -103,6 +103,11 @@ However, it is still important that your references use Title Case for article t
 You are responsible for checking that any special characters appear properly, and otherwise ensuring that your final reference list is correct and well-formatted, including sufficient detail.
 Pay particular attention to formatting of non-journal articles as it is easy to end up omitting key citation information for content like software, poster presentations, preprints or other materials.
 
+### Error analysis
+
+All original computational/numerical work presented in LiveCoMS is expected to include appropriate error analysis/error bars, and it should be clearly indicated how any error analysis was done and what quantities are presented (standard error in the mean, confidence interval (at what percentile), etc.)
```

Comment:
We could add, "Preliminary guidance can be obtained from the yet-to-be-peer-reviewed [Best Practices manuscript](https://github.com/dmzuckerman/Sampling-Uncertainty/) on quantifying uncertainty."  And we can change that to "Error analysis should conform to ..." once that article (or a suitable competitor) is published.

If you think it's premature for that, no worries on my end.

FYI, should be "standard error OF the mean"

Reply:
Yes on all of that. :)